frontend/src/components/chat/ChatInterface.tsx

```diff
@@ -52,6 +53,7 @@ function ChatInterface() {
   const dispatch = useDispatch();
   const { messages } = useSelector((state: RootState) => state.chat);
   const { curAgentState } = useSelector((state: RootState) => state.agent);
+  const [isMuted, setIsMuted] = useState(false);
```

Comment:
Please make sound off by default.
```suggestion
  const [isMuted, setIsMuted] = useState(true);
```